src/librehatti/catalog/lookups.py

```diff
@@ -26,7 +26,7 @@ def get_query(self, q, request):
         return user[0:15]
 
     def get_result(self, obj):
-        return unicode(obj.username)
+        return str(obj.username)
```

Comment:
Why not unicode here?

Reply:
Python 3 renamed the unicode type to str
https://stackoverflow.com/questions/19877306/nameerror-global-name-unicode-is-not-defined-in-python-3/49509744#49509744

Reply:
Ah, yes. Skipped my mind. Thanks.